Main.hs

```diff
@@ -2,24 +2,56 @@ module Main where
 
 import Control.Monad (unless)
 import Text.Printf (printf)
+import Data.List
+
+shorterThan :: Int -> [a] -> Bool
+shorterThan 0 [] = False
+shorterThan 0 xs = False
+shorterThan n [] = True
+shorterThan n (x:xs) = shorterThan (n-1) xs
 
 short :: [a] -> Bool
-short = undefined 
+short = shorterThan 3
+
+element :: Int -> [a] -> a
+element 0 (x:xs) = x
+element n [] = error "no element found"
+element n (x:xs) = element (n-1) xs
```

Comment:
There is a function in the Prelude that does exactly this: `(!!)` but with a different order of the arguments.